include/bitcoin/client/obelisk/obelisk_types.hpp

```diff
@@ -48,8 +40,10 @@ typedef std::vector<history_row> history_list;
 
 struct BCC_API stealth_row
 {
-    data_chunk ephemkey;
-    payment_address address;
+    // No sign byte in public key.
+    hash_digest ephemkey;
+    // No version byte in address.
+    short_hash address;
     hash_digest transaction_hash;
```

Comment:
because in the database I don't store that version byte. it simply scans all the rows returning results. If I wanted to get that byte, I would need another database to do a lookup or include that byte in the database. Since it isn't needed, I don't bother wasting space with lots of extraneous 00 per row. Imagine I'd need to fetch_transaction, find the output, get the address just to lookup 1 byte we don't really need.